### PR TITLE
backport: merge bitcoin#16521,18495,16929,18770,15760,19069,19630

### PR DESCRIPTION
**Pull request by vijaydasmp**

### CODE REVIEW

**Comment by vijaydasmp** [comment]
for bitcoin#18704 code changes present in upstream (https://github.com/bitcoin/bitcoin/commit/9ea4d8326a9d1641dfacca7ac82c16b7d387d967) file util.py is present in test/functional/test_framework/test_framework.py in dash code, appropriately updated def sync_blocks and def sync_mempools functions for dash

**Comment by vijaydasmp** [comment]
Hello @UdjinM6

**Review by UdjinM6 — DISMISSED** [review]
LGTM, utACK

**Comment by vijaydasmp** [comment]
Hello @PastaPastaPasta

**Comment by vijaydasmp** [comment]
Hello @UdjinM6 , please have a look

File: src/policy/fees.h [hunk 1]
```diff
@@ -145,9 +145,9 @@ class CBlockPolicyEstimator
 
     /** Decay of .962 is a half-life of 18 blocks or about 45 minutes */
     static constexpr double SHORT_DECAY = .962;
-    /** Decay of .998 is a half-life of 144 blocks or about 6 hours */
+    /** Decay of .9952 is a half-life of 144 blocks or about 1 day */
```
Comment (UdjinM6) [inline src/policy/fees.h:148]: 19630:
```suggestion
    /** Decay of .9952 is a half-life of 144 blocks or about 6 hours */
```

File: src/policy/fees.h [hunk 2]
```diff
@@ -145,9 +145,9 @@ class CBlockPolicyEstimator
...
     static constexpr double MED_DECAY = .9952;
-    /** Decay of .9995 is a half-life of 1008 blocks or about 2 days */
+    /** Decay of .99931 is a half-life of 1008 blocks or about 1 week */
```
Comment (UdjinM6) [inline src/policy/fees.h:150]: 19630:
```suggestion
    /** Decay of .99931 is a half-life of 1008 blocks or about 2 days */
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
one tiny issue, LGTM otherwise

File: src/policy/fees.h (hunk 2 above)
Comment (vijaydasmp) [reply src/policy/fees.h:150]: Thanks

File: src/policy/fees.h (hunk 1 above)
Comment (UdjinM6) [reply src/policy/fees.h:148]: this one is not fixed

Comment (vijaydasmp) [reply src/policy/fees.h]: Oh!, now fixed

**Review by UdjinM6 — APPROVED** [review]
LGTM, utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

**Pull request merged**